active/InformationBar/InformationBar.md

```diff
@@ -0,0 +1,536 @@
+# Background
+> This spec corresponds to [issue 913](https://github.com/microsoft/microsoft-ui-xaml/issues/913) on the WinUI repo.
+
+Users should be informed about essential status changes that occur on an app level.
```

Comment:
Would it be easy to add a couple of pictures?

Reply:
Do you mean throughout the document or in the Background section specifically? There should be pictures/mockups throughout the spec...